pkgs/args/lib/src/usage.dart

```diff
@@ -90,10 +90,8 @@ class _Usage {
     if (option.help != null) _write(2, option.help!);
 
     if (option.allowedHelp != null) {
-      var allowedNames = option.allowedHelp!.keys.toList();
-      allowedNames.sort();
       _newline();
-      for (var name in allowedNames) {
+      for (var name in option.allowedHelp!.keys) {
```

Comment:
nit: consider an if case to extract this as a variable and avoid the `!`

Reply:
Done, did a couple nearby ones as well.